python-package/README.rst

```diff
@@ -294,4 +298,3 @@ To check that a contribution to the package matches its style expectations, run
    :target: https://pepy.tech/project/lightgbm
 .. |API Docs| image:: https://readthedocs.org/projects/lightgbm/badge/?version=latest
    :target: https://lightgbm.readthedocs.io/en/latest/Python-API.html
-.. _CMake: https://cmake.org/
```

Comment:
Remove CMake version from here as well?
https://github.com/microsoft/LightGBM/blob/2bc3ab86b78e6904da49a1053219695f83c2d21e/R-package/README.md?plain=1#L145

Reply:
Ah yes, missed this one! Removed in https://github.com/microsoft/LightGBM/pull/6536/commits/8cee2d623cf54823e0c88de37044f64051ca54e2